app/models/answer.js

```diff
@@ -1,41 +1,43 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    config = require('../../config/config'),
-    Schema = mongoose.Schema;
+const mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
 
 /**
  * Answer Schema
  */
-var AnswerSchema = new Schema({
-    id: {
-        type: Number
-    },
-    text: {
-        type: String,
-        default: '',
-        trim: true
-    },
-    official: {
-        type: Boolean
-    },
-    expansion: {
-        type: String,
-        default: '',
-        trim: true
-    }
+const AnswerSchema = new Schema({
+  id: {
+    type: Number
+  },
+  text: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  official: {
+    type: Boolean
+  },
+  expansion: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  location: {
+    type: String
+  }
 });
 
 /**
  * Statics
  */
 AnswerSchema.statics = {
-    load: function(id, cb) {
-        this.findOne({
-            id: id
-        }).select('-_id').exec(cb);
-    }
+  load(id, cb) {
```

Comment:
What exactly is 'cb',please use a more descriptive variable name (this occurs in other areas).